src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static StorageDao defaultStorage;
+    private static RegistrationServiceImpl registrationService;
+    private static final String NOT_EXIST_LOGIN = "forth";
+
+    @BeforeAll
+    static void beforeAll() {
+        defaultStorage = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        User firstUser = new User();
+        firstUser.setId(1L);
+        firstUser.setLogin("first");
+        firstUser.setPassword("111111");
+        firstUser.setAge(18);
+        User secondUser = new User();
+        secondUser.setId(2L);
+        secondUser.setLogin("second");
+        secondUser.setPassword("222222");
+        secondUser.setAge(19);
+        User thirdUser = new User();
+        thirdUser.setId(3L);
+        thirdUser.setLogin("third");
+        thirdUser.setPassword("333333");
+        thirdUser.setAge(20);
+        Storage.people.add(0, firstUser);
+        Storage.people.add(1, secondUser);
+        Storage.people.add(2, thirdUser);
+    }
+
+    @Test
+    void check_userNotExist_ok() {
```

Comment:
<img width="816" alt="Screen Shot 2022-08-17 at 6 18 40 PM" src="https://user-images.githubusercontent.com/90692308/185178007-750c766c-f81e-4e8e-9a96-bd8da8432f2f.png">

All names of your test methods must end with ok if the result is positive or notOk if it is negative. Also think about the names of your methods, their names should be clear what I'm testing. Fix it everywhere